samples/fsharp/getting-started/BinaryClassification_SentimentAnalysis/Program.fs

```diff
@@ -1,123 +1,141 @@
-﻿module BinaryClassification_SentimentAnalysis
+﻿open System
 
-open System
+open Microsoft.ML.Runtime.Learners
+open Microsoft.ML.Runtime.Data
+open Microsoft.ML
 open System.IO
 
-open Microsoft.ML.Runtime.Api;
-open Microsoft.ML.Legacy;
-open Microsoft.ML.Legacy.Models;
-open Microsoft.ML.Legacy.Data;
-open Microsoft.ML.Legacy.Transforms;
-open Microsoft.ML.Legacy.Trainers;
 
-[<CLIMutable>]
-type SentimentData =
-    { [<Column("0")>]
-      SentimentText: string
-      [<Column("1", name="Label")>]
-      Sentiment: float32 }
 
-[<CLIMutable>]
-type SentimentPrediction =
-    { [<ColumnName("PredictedLabel")>]
-      Sentiment: bool }
+type [<CLIMutable>] SentimentIssue = { Label : bool; Text : string }
```

Comment:
I think beginners prefer to see plenty of whitespace, so format these over multiple lines with a line whitespace in between.  Also add a `///` comment to both